masschallenge/django-accelerator
# [AC-9459]: Correct typos on the Business Proposition help text

[AC-9459](https://masschallenge.atlassian.net/browse/AC-9459)

How to test:
- Confirm that all typos has been fixed on http://localhost:8181/startup/20/business-proposition/

Sibling PR: https://github.com/masschallenge/accelerate/pull/3229
